ext/reflection/php_reflection.c

```diff
@@ -7082,7 +7082,10 @@ ZEND_METHOD(ReflectionFiber, getExecutingLine)
 		prev_execute_data = fiber->execute_data->prev_execute_data;
 	}
 
-	RETURN_LONG(prev_execute_data->opline->lineno);
+	if (prev_execute_data->func && ZEND_USER_CODE(prev_execute_data->func->common.type)) {
```

Comment:
I wonder if we should look for the user parent call when this is an internal function? This was not specified by the RFC. @trowski What's your opinion on this?

Reply:
I think that would make sense where possible, yes.

I don't think that would work for the `$f = new Fiber(Fiber::suspend(...));` case, so we probably still need to change the return type.

Reply:
So should I change this to return info about any userland parent call (if at all present, returning null otherwise), or should I leave it as it is?

Reply:
If you could make the change that would be great :slightly_smiling_face: